rpcs3/rpcs3qt/settings_dialog.cpp

```diff
@@ -963,8 +963,16 @@ settings_dialog::settings_dialog(std::shared_ptr<gui_settings> gui_settings, std
 	m_emu_settings->EnhanceCheckBox(ui->debugConsoleMode, emu_settings::DebugConsoleMode);
 	SubscribeTooltip(ui->debugConsoleMode, tooltips.settings.debug_console_mode);
 
-	m_emu_settings->EnhanceCheckBox(ui->accuratePPUfma, emu_settings::AccuratePPUfma);
-	SubscribeTooltip(ui->accuratePPUfma, tooltips.settings.accurate_ppu_fma);
+	if (utils::has_fma3() || utils::has_fma4())
+	{
+	ui->accuratePPUfma->setEnabled(false);
+	SubscribeTooltip(ui->accuratePPUfma, tr("Improves FMA operations' accuracy at the cost of performance.\nWhile disabling it might give a decent performance boost, it may also introduce subtle bugs that otherwise do not occur."));
```

Comment:
indentation.

If there's a tooltip class.... why don't you use it?

Reply:
and why is there a long tooltip for a disabled checkbox?

Reply:
Copypasta 🤦‍♀️ will fix.

Reply:
Should be resolved now. Please review.